test/pseudo-tty/test-stderr-stdout-handle-sigwinch.out

```diff
@@ -1,2 +1,2 @@
-calling stdout._refreshSize
```

Comment:
The order changed here because in the test `test-stderr-stdout-handle-sigwinch`, `process.stderr` is accessed before `process.stdout` so the SIGWINCH even listeners are attached in this order. I don't think the order by which we resize the two streams makes a difference, though.